docs/pages/deploy-a-cluster/linux-demo.mdx

```diff
@@ -0,0 +1,208 @@
+---
+title: Run a Self-Hosted Demo Cluster 
+description: This tutorial will guide you through the steps needed to install and run Teleport on a Linux server
+videoBanner: BJWbSqiDLeU
+tocDepth: 3
+---
+
+See how a self-hosted Teleport deployment works by completing the tutorial
+below. This shows you how to spin up a single-instance Teleport cluster on a
```

Comment:
```suggestion
below. This shows you how to spin up a single instance Teleport cluster on a
```

Reply:
This is a compound modifier, so I think the hyphen is okay here.